packages/components/atoms/f-card/src/components/_tests/Card.test.js

```diff
@@ -210,4 +210,24 @@ describe('Card', () => {
             });
         });
     });
+
+    describe.each([
+        [true, true],
+        [false, false]
+    ])('`hasFullWidthBottomElement` is %s', (passedValue, expected) => {
+        // Arrange
+        const propsData = {
+            hasFullWidthBottomElement: passedValue
+        };
+
+        // Act
+        const wrapper = shallowMount(Card, { propsData });
+
+        // Act
+        const testedElement = wrapper.find('[data-test-id="card-component-fullWidthBottomElement"]');
+
+        it(`should ${wrapper.vm.hasFullWidthBottomElement ? '' : 'not '}display a bottom positioned full width element`, () => {
+            expect(testedElement.exists()).toBe(expected);
+        });
+    });
```

Comment:
I think it might be more readable to have the following format. But I don't have a strong opinion on this. So ignore this comment if you think otherwise.

```
   describe('hasFullWidthBottomElement', () => {
       it.each([
           [true, true],
           [false, false]
       ])('`should return "%s" when `hasFullWidthBottomElement` is "%s"', (passedValue, expected) => {
           ....
       }
   });
```

Reply:
The outcome at the moment says: "Card › `hasFullWidthBottomElement` is true › should display a bottom positioned full width element"

I will change to "Card › if `hasFullWidthBottomElement` prop is true › should display a bottom positioned full width element"